analysed-packages/logrotate/version-3.20.1/logrotate-3.20.1-SPDX2TV.spdx

```diff
@@ -0,0 +1,4013 @@
+LicenseInfoInFile: LicenseRef-FSFULLR
+LicenseInfoInFile: LicenseRef-FSFULLRWD
+LicenseInfoInFile: LicenseRef-fsf-unlimited-no-warranty
+FileCopyrightText: <text> Copyright (C) 1994-2021 Free Software Foundation, Inc. </text>
+
+##File
+
+FileName: logrotate-3.20.1.tar.xz/logrotate-3.20.1/aclocal.m4
+SPDXID: SPDXRef-item547312
+FileChecksum: SHA1: ca535c4c32cb9c6f44d9451f467ee39993ac5559
+FileChecksum: SHA256: ba80976d554f87a70b1e64f913152bf7b36da1d08849a7e58e457bd393c4d20e
+FileChecksum: MD5: 18205747061a2a9def8201b627af0e6b
+LicenseConcluded: LicenseRef-FSFULLR AND LicenseRef-FSFULLRWD
+LicenseInfoInFile: LicenseRef-FSFULLR
+LicenseInfoInFile: LicenseRef-FSFULLRWD
+LicenseInfoInFile: LicenseRef-fsf-unlimited-no-warranty
+LicenseInfoInFile: LicenseRef-fsf-unlimited
+FileCopyrightText: <text> Copyright (C) 1996-2021 Free Software Foundation, Inc.
+Copyright (C) 2002-2021 Free Software Foundation, Inc.
+Copyright (C) 2001-2021 Free Software Foundation, Inc.
+Copyright (C) 1997-2021 Free Software Foundation, Inc.
+Copyright (C) 1999-2021 Free Software Foundation, Inc.
+Copyright (C) 1999-2021 Free Software Foundation, Inc.
+Copyright (C) 1996-2021 Free Software Foundation, Inc.
+Copyright (C) 2001-2021 Free Software Foundation, Inc.
+Copyright (C) 2003-2021 Free Software Foundation, Inc.
+Copyright (C) 2001-2021 Free Software Foundation, Inc.
+Copyright (C) 1997-2021 Free Software Foundation, Inc.
+Copyright (C) 2001-2021 Free Software Foundation, Inc.
+Copyright (C) 1999-2021 Free Software Foundation, Inc.
+Copyright (C) 2001-2021 Free Software Foundation, Inc.
+Copyright (C) 1996-2021 Free Software Foundation, Inc.
+Copyright (C) 2009-2021 Free Software Foundation, Inc.
+Copyright (C) 2001-2021 Free Software Foundation, Inc.
+Copyright (C) 2006-2021 Free Software Foundation, Inc.
+Copyright (C) 2004-2021 Free Software Foundation, Inc. </text>
+
+##File
+
+FileName: logrotate-3.20.1.tar.xz/logrotate-3.20.1/compile
+SPDXID: SPDXRef-item547302
+FileChecksum: SHA1: c37f7649d459d82714935cdd98c686763351f12e
+FileChecksum: SHA256: c207b390aac6323062b982214a6c63448e53e6911107993abe96f35fe7a30a18
+FileChecksum: MD5: b816f92498a6ec8138d9ec53a14bcaa4
+LicenseConcluded: LicenseRef-GPL-2.0-or-later-WITH-Autoconf-exception-generic
+LicenseInfoInFile: LicenseRef-GPL-2.0-or-later
+LicenseInfoInFile: LicenseRef-Autoconf-exception
+LicenseInfoInFile: LicenseRef-Autoconf-exception-generic-3.0
+LicenseInfoInFile: LicenseRef-gpl-2.0-plus
+LicenseInfoInFile: LicenseRef-autoconf-simple-exception-2.0
+FileCopyrightText: <text> Copyright (C) 1999-2021 Free Software Foundation, Inc. Written by Tom Tromey <tromey@cygnus.com>. </text>
+
+##File
+
+FileName: logrotate-3.20.1.tar.xz/logrotate-3.20.1/configure
+SPDXID: SPDXRef-item547084
+FileChecksum: SHA1: ed9b653d59eeae0a6d53478f29566f72922e422b
+FileChecksum: SHA256: 4eadb229fe88f96bcb5241025681a8ec27a438311d5b216f378eb071167bc2a7
+FileChecksum: MD5: 05d3ddaaffccf560e3bd69d6dd3e8c1a
+LicenseConcluded: LicenseRef-FSFUL
```

Comment:
Please add additionally to the license text:
This configure script is free software; the Free Software Foundation
gives unlimited permission to copy, distribute and modify it.
The following text:
This config.status script is free software; the Free Software Foundation
gives unlimited permission to copy, distribute and modify it.
This is near the end of the file